docs/source/tutorials/advanced_users.md

```diff
@@ -123,8 +121,7 @@ quantization_config = {
     "weight_type" : "QUInt8"
 }
 # search over the values for the other config parameters
-quantization_pass = create_pass_from_dict(OnnxQuantization, quantization_config)
-engine.register(quantization_pass)
+engine.register(OnnxQuantization, quantization_pass, False)
```

Comment:
I think this is `quantization_config` instead of `quantization_pass` here?

Reply:
good catch